phantazm-zombies/src/main/java/com/github/phantazmnetwork/zombies/equipment/gun/target/tester/PhantazmTargetTester.java

```diff
@@ -16,10 +17,20 @@
 import java.util.Objects;
 import java.util.UUID;
 
+/**
+ * A {@link TargetTester} that only selects {@link PhantazmMob}s.
+ */
 public class PhantazmTargetTester implements TargetTester {
```

Comment:
PhantazmMobTargetTester (verbose but PhantazmTargetTester sounds like it's the canonical TargetTester for our project)